src/rewrites/rewrite_http_to_https.conf

```diff
@@ -6,8 +6,20 @@
 #
 # https://wiki.apache.org/httpd/RewriteHTTPToHTTPS
 
+# (1) If you're using cPanel AutoSSL or the Let's Encrypt webroot
+#     method it will fail to validate the certificate if validation 
+#     requests are redirected to HTTPS. Turn on the condition(s) 
+#     you need.
+#
+#     https://www.iana.org/assignments/well-known-uris/well-known-uris.xhtml
+#     https://tools.ietf.org/html/draft-ietf-acme-acme-12
+
 <IfModule mod_rewrite.c>
    RewriteEngine On
    RewriteCond %{HTTPS} !=on
+   # (1)
+   # RewriteCond %{REQUEST_URI} !^/\.well-known/acme-challenge/[0-9a-zA-Z_-]+$
+   # RewriteCond %{REQUEST_URI} !^/\.well-known/cpanel-dcv/[0-9a-zA-Z_-]+$
```

Comment:
Are you sure these both regular expressions cannot be simplified?
Maybe `[\d-]` instead of `[0-9a-zA-Z_-]`? (I'm not sure that said...)